R/lag_valideringsdatasett.R

```diff
@@ -71,15 +71,28 @@ lag_valideringsdatasett = function(d_reg, indvars) {
   datavars = vars %>%
     setdiff(indvars)
 
-  # Lag oversikt over variabeltypar
-  vartypar = map(d_reg[datavars], ~ class(.x)) %>%
-    map_chr(first) %>%
-    unname()
-  unike_vartypar = unique(vartypar)
+
+  d_vartypar = tibble::tibble(
+    vartypar = map(d_reg[datavars], ~ class(.x)) %>%
+      unname(), # Lag oversikt over variabeltypar
+
+    vartypar_sammenlagt = map_chr(
+      vartypar,
+      \(x) paste(x,
+        collapse = "_"
+      )
+    ) # Slå sammen elementer fra samme klass i en string
```

Comment:
Ev. `# Slå sammen alle (under)klassenavn til én tekststreng`.
Men eigentleg er det vel ikkje nødvendig med ein kommentar i det heile tatt?

Reply:
Gjerne la den stå for tiden.

Reply:
OK.